### DISCUSSION TITLE
Your bootstrap/package.json repeats one section

### DISCUSSION THREAD

**Discussion by darrenaddy** [twbs/bootstrap#36274] [General] [answered]
Just an FYI:
`"dependencies": {},
  "peerDependencies": {
    "@popperjs/core": "^2.9.2"
  },`
is found on lines 98-101 and then again in the final lines of the document.

**Comment by mdo** [comment, marked as answer]
The repeated section is for JSPM, as it's indented under that section. Check the git blame history for more info.
